osu.Game/Screens/OnlinePlay/Components/MatchBeatmapDetailArea.cs

```diff
@@ -43,9 +44,10 @@ public MatchBeatmapDetailArea()
                             {
                                 RelativeSizeAxes = Axes.Both,
                                 Padding = new MarginPadding { Bottom = 10 },
-                                Child = playlist = new DrawableRoomPlaylist(true, false)
+                                Child = playlist = new PlaylistsRoomSettingsPlaylist
                                 {
                                     RelativeSizeAxes = Axes.Both,
+                                    AllowSelection = true,
```

Comment:
This component is used in playlists song select, and it didn't use to support selection (see old code - `allowSelection`, the second ctor param, was `false`). Any reason to allow it now?